src/main/app.ts

```diff
@@ -290,6 +290,11 @@ export class KandoApp {
       return true;
     }
 
+    if (options.closeMenu) {
+      this.menuWindow?.webContents.send('menu-window.hide-menu');
```

Comment:
This would be better:


```suggestion
      this.menuWindow?.hide();
```